v0/destinations/marketo/util.js

```diff
@@ -13,7 +13,15 @@ const { TRANSFORMER_METRIC } = require("../../util/constant");
 const ErrorBuilder = require("../../util/error");
 
 const MARKETO_RETRYABLE_CODES = ["601", "602", "604", "611"];
-const MARKETO_ABORTABLE_CODES = ["600", "603", "605", "609", "610", "612"];
+const MARKETO_ABORTABLE_CODES = [
```

Comment:
Can we add some documentation links from the destination for these error types if available?

Reply:
Sure

Reply:
Added the doc. Please check

Reply:
You provided the wrong link. Can you check if [this](https://developers.marketo.com/rest-api/error-codes/) link is suitable?

Reply:
Sorry. Corrected the link now.